fr_FR.json

```diff
@@ -0,0 +1,798 @@
+    "OUTFIT_JBIB": "JBIB",
+    "OUTFIT_GLASSES": "GLASSES",
+    "OUTFIT_EARS": "EARS",
+    "OUTFIT_UNK1": "UNK1",
+    "OUTFIT_UNK2": "UNK2",
+    "OUTFIT_UNK3": "UNK3",
+    "OUTFIT_WATCH": "WATCH",
+    "OUTFIT_WRIST": "WRIST",
+    "OUTFIT_UNK4": "UNK4",
+    "OUTFIT_RANDOM_COMPONENT": "Random Component",
+    "OUTFIT_DEFAULT_COMPONENT": "Default Component",
+    "OUTFIT_RANDOM_PROPS": "Random Props",
+    "OUTFIT_CLEAR_PROPS": "Clear Props",
+    "OUTFIT": "Outfit",
+    "EXPORT_TO_CLIPBOARD": "Export To Clipboard",
+    "IMPORT_FROM_CLIPBOARD": "Import From Clipboard",
+    "OUTFIT_TEX": "TEX",
+    "OUTFIT_SAVE_CURRENT": "Save Current",
+    "OUTFIT_APPLY_SELECTED": "Apply Selected",
+    "OUTFIT_DELETE_SELECTED": "Delete Selected",
+    "OUTFIT_SLOT": "Slot",
+    "OUTFIT_NAME": "Name",
+    "GUI_TAB_ORBITAL_DRONE": "Orbital Drone",
+    "ORBITAL_DRONE_USAGE_DESCR": "press 'Look behind' (C/R3 by default) to activate\npress WASD buttons or Left thumbstick to navigate\nUse scroll wheel/Mouse or Right thumbstick to zoom\npress E/Q or L1/R1 to modify speed\npress 'Jump' (Space/X/Square by default) to lock on an entity\npress 'Fire' (Mouse button 1/Right trigger by default) to Obliterate\npress Enter or A/X by default to teleport to target",
+    "ORBITAL_DRONE_AUTO_LOCK_ON_PLAYER": "Automatically lock onto player",
+    "ORBITAL_DRONE_AUTO_LOCK_ON_PLAYER_TOOLTIP": "If enabled, changes the selected player to the one it detects upon locking on an entity\nAll explosions will be blamed on the selected player, defaulting to the local player",
+    "ORBITAL_DRONE_HIGH_SPEED_MULTIPLIER": "High speed orbital drone multiplier:",
+    "ORBITAL_DRONE_LOW_SPEED_MULTIPLIER": "Slow speed orbital drone multiplier:",
+    "VEHICLE_CONTROLLER": "Vehicle Controller",
+    "VEHICLE_CONTROLLER_TRY_ALT_PATHFINDING": "Failed to pathfind using accurate method, using alternative method.",
+    "VEHICLE_CONTROLLER_FOUND_LOCATION": "Found a nice spot, your vehicle is on its way.",
+    "VEHICLE_CONTROLLER_FORCE_PATHFINDING": "Couldn't locate an accurate spot, your vehicle is on its way regardless.",
+    "VEHICLE_CONTROLLER_NAVMESH_FAILURE": "Failed to load navmesh, unable to navigate.",
+    "OPEN": "Open",
+    "CLOSE_ALL_DOORS": "Close All",
+    "OPEN_ALL_DOORS": "Open All",
+    "VEHICLE_CONTROLLER_ALL_DOORS": "All doors",
+    "VEHICLE_CONTROLLER_TOGGLE_LIGHTS": "Toggle Lights",
+    "VEHICLE_CONTROLLER_TOGGLE_HIGH_BEAMS": "Toggle High beams",
+    "VEHICLE_CONTROLLER_INTERIOR_LIGHTS_ON": "Interior lights on",
+    "VEHICLE_CONTROLLER_INTERIOR_LIGHTS_OFF": "Interior lights off",
+    "VEHICLE_CONTROLLER_NEON_LIGHTS": "Neon lights",
+    "VEHICLE_CONTROLLER_ENTER_VEHICLE_SPEED_WALKING": "Walk",
+    "VEHICLE_CONTROLLER_ENTER_VEHICLE_SPEED_RUNNING": "Run",
+    "VEHICLE_CONTROLLER_ENTER_VEHICLE_SPEED_SPRINTING": "Sprint",
+    "POTENTIAL_ADMIN_FOUND": "Potential Admin Found!",
+    "PLAYER_DETECTED_AS_ADMIN": "{} has been detected as admin"
+}
```

Comment:
These lines don't appear to have been translated yet.

Reply:
Fuck wrong one sur